### ISSUE TITLE
Provide feedback to the user while selecting cells for table inspector

### ISSUE THREAD

**Issue by rajatvijay** [centerofci/mathesar#1636]
As a user, I want to see what cells will be selected while selecting them something similar to Google Sheets. As of now only once the selection is complete do I see the blue background over the selected cells.

**Comment by seancolsen** [comment]
Duplicate of  #1623
